Update jsoc admins list (#2021)
JuliaLang/www.julialang.org: jsoc/admins.md

The JSoC (encompasses GSoC, GSoD, etc.) program admins have a few responsibilities including, but not limited too, reviewing contributor applications for all programs, creating and submitting the organization application for each program, helping ensure the programs run smoothly, recruiting mentors, making final contributor selections and more. 

The current JSoC admins are: 

> - [Avik Sengupta](https://filebox.ece.vt.edu/~aviksg/) - Julia Project Lead at NumFOCUS

Reviewer: @aviks is that you? 🤔

Reviewer 2: Probabky best to keep the original link 😆

Author: ```suggestion
- [Avik Sengupta](https://twitter.com/aviksengupta) - Julia Project Lead at NumFOCUS
```
Oops, thanks!

The current JSoC admins are: 
- [Avik Sengupta](https://twitter.com/aviksengupta) - Julia Project Lead at NumFOCUS

> - [Jacob Scott Zelko](https://jacobzelko.com/) - Prominent contributor to the [JuliaHealth](https://juliahealth.org/) ecosystem

Reviewer 3: If it is alright with @DilumAluthge (and all other parties I may not be thinking of), maybe put me here as:

"Current lead of the [JuliaHealth](https://juliahealth.org/) ecosystem"

Otherwise, looks great! Many thanks @LilithHafner :pray:

Author: ```suggestion
- [Jacob Scott Zelko](https://jacobzelko.com/) - Current lead of the [JuliaHealth](https://juliahealth.org/) ecosystem
```

Thanks for providing a more accurate characterization of your role :)
